beesdoo_base/security/groups.xml

```diff
@@ -2,6 +2,6 @@
 <odoo>
     <record id="group_force_barcode" model="res.groups">
         <field name="name">Bees Card Force Barcode</field>
-        <field name="users" eval="[(6, 0, [ref('base.user_root')])]"/>
+        <field name="users" eval="[(4, ref('base.user_root'))]"/>
```

Comment:
Shouldn't it be `[(4, ref('base.user_root'), False)]`  (triplet).

https://www.odoo.com/documentation/12.0/reference/orm.html

Reply:
C'est optionnel:
> The _ underscore symbol used in the preceding list represents irrelevant values, usually
filled with 0 or False .The trailing irrelevant values can be safely omitted. For example, (4, id,_) can be used as (4, id).
source : odoo 12 development essentials, p.148-149

Reply:
then 👌